assets/css/amp-editor-story-blocks.css

```diff
@@ -364,41 +530,114 @@ div[data-type="amp/amp-story-page"] .editor-inner-blocks .editor-block-list__lay
 		Content - 383px (including 15px padding)
 		Block Navigation - <= 300px
 	*/
+	:root {
+		--block-settings-sidebar-width: 280px;
+		--admin-menu-expanded-sidebar-width: 160px;
+		--admin-menu-collapsed-sidebar-width: 36px;
+		--amp-story-content-width: 338px;
+		--block-editor-horizontal-margin: 40px;
+		--scrollbar-width: 20px;
+		--block-navigation-right-margin: 20px;
+	}
 
 	/* Navigator in the context of the open sidebar, settings hidden. */
-	.post-type-amp_story .edit-post-layout #amp-root-navigation {
-		max-width: calc( 100vw - 160px - 383px );
+	.post-type-amp_story:not(.folded) .edit-post-layout #amp-root-navigation {
+		max-width: calc(
+			calc(
+				100vw
+				-
+				var(--admin-menu-expanded-sidebar-width)
+				-
+				var(--amp-story-content-width)
+				-
+				calc( var(--block-editor-horizontal-margin) * 2 )
+				-
+				var( --scrollbar-width )
+			) / 2
+			-
+			var(--block-navigation-right-margin)
+		);
+	}
+
+	.post-type-amp_story.folded .edit-post-layout #amp-root-navigation {
+		max-width: calc(
+			calc(
+				100vw
+				-
+				var(--admin-menu-collapsed-sidebar-width)
+				-
+				var(--amp-story-content-width)
+				-
+				calc( var(--block-editor-horizontal-margin) * 2 )
+				-
+				var( --scrollbar-width )
+			) / 2
+			-
+			var(--block-navigation-right-margin)
+		);
 	}
 
 	.post-type-amp_story .edit-post-layout .editor-writing-flow > div > div {
-		grid-template-columns: auto minmax( 368px, calc( 100vw - 160px - 315px ) );
+		grid-template-columns: auto minmax( 368px, calc( 100vw - var(--admin-menu-expanded-sidebar-width) - 315px ) ); /* @todo Replace 368px and 315px with variables. */
```

Comment:
@swissspidy Would you please add variables for `368px` and `315px`? I wasn't sure what magic they referred to, as they aren't mentioned in the comment above. See other instances of these numbers below.

Reply:
They refer to the page individual sizes, but this was before my time. I don't think the grid here is needed anymore or even applies properly. Will see if I can remove it.